core/coveredCall.js

```diff
@@ -14,7 +14,7 @@ import { mustBeSameStructure, sameStructure } from '../util/sameStructure';
  * limited, and the offerer and potential acceptor are 'bob' and 'alice'
  * respectively.
  */
-const coveredCall = {
+const coveredCall = harden({
```

Comment:
I think (and I could be wrong here) that we want to harden things that are exported for the module to be pure.

Reply:
> The only way coveredCall escapes is in the exported sources. Should our style be to harden() everything, even if it won't be allowed to escape?

Just about. When in doubt, harden.

> I think (and I could be wrong here) that we want to harden things that are exported for the module to be pure.

Yes. We will want to harden enough that the Jessie static checker can verify that the module would be pure if its imports are pure. Only avoid hardening when you're confident that the Jessie static checker would be ok with it. When in doubt, harden.

There are likely to arise special cases where we need not to harden. These should align with special cases where we need to leave Jessie and program in non-Jessie SES. It would be good to comment each of those cases explicitly.